lib/jekyll/site.rb

```diff
@@ -73,11 +73,12 @@ def process
       render
       cleanup
       write
-      print_stats
+      print_stats if config["profile"]
     end
 
     def print_stats
-      if @config["profile"]
+      # TODO: remove duplicate guard statement in next major version change
+      if config["profile"]
```

Comment:
Let's remove this line – I will tell anyone who complains that it was a bug to have this check in this method; if I call print_stats, it had better print stats! 😉